ansible_builder/main.py

```diff
@@ -298,7 +311,8 @@ def __init__(self, definition,
         self.container_runtime = container_runtime
         self.tag = tag
         self.steps = [
-            "FROM {0} as galaxy".format(self.base_image),
+            "ARG BASE_IMAGE={}".format(self.base_image),
```

Comment:
maybe ANSIBLE_RUNNER_IMAGE, if we want to be specific that it should be some sort of ansible-runner based image.

Reply:
Sure. I've pushed that up.